site/content/installation/ngf-images/building-the-images.md

```diff
@@ -57,14 +57,14 @@ If building the NGINX Plus image, you will also need a valid NGINX Plus license
      make PREFIX=myregistry.example.com/nginx-gateway-fabric build-nginx-image
      ```
 
-   - To build just the NGINX Plus image:
+   - To build just the NGINX Plus image, replace `<my-docker-registry>` with your registry's path:
 
      ```makefile
-     make PREFIX=myregistry.example.com/nginx-gateway-fabric build-nginx-plus-image
+     make PREFIX=<my-docker-registry>/nginx-gateway-fabric/nginx-plus/nginx-plus build-nginx-plus-image
      ```
 
    Set the `PREFIX` variable to the name of the registry you'd like to push the image to. By default, the images will be
-   `ghcr.io/nginxinc/nginx-gateway-fabric/nginx:edge` or `ghcr.io/nginxinc/nginx-gateway-fabric/nginx-plus:edge`.
+   `nginx-gateway-fabric/nginx:edge` or `ghcr.io/nginxinc/nginx-gateway-fabric/nginx-plus:edge`.
```

Comment:
ghcr.io prefix should be removed from the nginx-plus image here as well.

Reply:
removing